DesignSafe-CI/portal
# task/DES-1917: Update local certs

## Overview: ##

Local certs expired

## PR Status: ##


## Related Jira tickets: ##

* [DES-1917](https://jira.tacc.utexas.edu/browse/DES-1917)

## Testing Steps: ##
1. Follow the steps in the [README](https://github.com/DesignSafe-CI/portal#installing-local-ca) to accept the new self signed cert
2. Go to https://designsafe.dev and confirm the new cert works as expect and is valid

## UI Photos:
<img width="500" alt="Screen Shot 2021-04-01 at 2 19 37 PM" src="https://user-images.githubusercontent.com/20326896/113343345-52e03400-92f5-11eb-9310-7a688c4446dc.png">
